monitor/services/src/main/java/org/datacleaner/monitor/configuration/ConfigurationCache.java

```diff
@@ -106,8 +107,12 @@ protected DataCleanerConfiguration decorateConfiguration(DataCleanerConfiguratio
     }
 
     protected DataCleanerConfiguration readConfiguration() {
+        return readConfiguration(null);
+    }
+    
+    DataCleanerConfiguration readConfiguration(Map<String, String> overrideProperties) {
         final ConfigurationReaderInterceptor interceptor = new MonitorConfigurationReaderInterceptor(_repository,
-                _tenantContext, _injectionManagerFactory);
+                _tenantContext, overrideProperties, _injectionManagerFactory);
```

Comment:
When the readConfiguration method is invoked from the ConfigurationCache itself, the value for the overrideProperties is always null. It only contains an object when this method is invoked directly from the TenantContextImpl (and the result is not cached). An other option would be to create a copy of this method on the TenantContextImpl class, but that would involve some more refactoring on that class so I have access to the InjectionManagerFactory.